src/assertions/OrderAssertions.ts

```diff
@@ -59,14 +59,39 @@ export class OrderAssertions {
     const issuanceOrder: IssuanceOrder = _.omit(signedIssuanceOrder, 'signature');
     const orderHash = SetProtocolUtils.hashOrderHex(issuanceOrder);
 
+    const {
```

Comment:
One thing to improve code readability is having logical groupings of statements. I'd have these declarations right below the `issuanceOrder` declaration, and group the `orderHash` declaration with the assertions that are called.